subsystems/climbersubsystem.py

```diff
@@ -0,0 +1,55 @@
+from wpilib import (DoubleSolenoid,
+    PneumaticsModuleType,
+    Solenoid, SmartDashboard)
+from commands2 import Subsystem
+from enum import Enum, auto
+import constants
+from util.simtalon import Talon
+
+
+class ClimberSubsystem(Subsystem):
+    class ClimberState(Enum):
+        ClimberRetract = auto()
+        ClimberExtend = auto()
+        ClimberNeutral = auto()
+        
+    def __init__(self):
+     Subsystem.__init__(self)
+     self.setName(__class__.__name__)
+    
+     self.climberMotor = Talon(
+            constants.kClimberCANID,
+            constants.kClimberName,
+            constants.kClimberPGain,
+            constants.kClimberIGain,
+            constants.kClimberDGain,
+            constants.kClimberInverted 
+        )
+     
+     self.climberMotor.setCurrentLimit(
+         
+     )
+     self.state = ClimberSubsystem.ClimberState.ClimberNeutral
+
+
+            
+    def periodic(self) -> None:
```

Comment:
also needed is logging of where the climber hook exists in space as a 3d pose for sim